docs/Explanation/presentations.md

```diff
@@ -0,0 +1,19 @@
+# Presentations
+
+## Introduction
```

Comment:
```suggestion
```

Reply:
we can omit this heading, wdyt? We strive to avoid stacked headings so let's just let the introduction to the presentations folder by the next sentence, no need to nest under Introduction